libraries/AP_RSSI/AP_RSSI.cpp

```diff
@@ -128,6 +128,13 @@ float AP_RSSI::read_receiver_rssi()
         case RssiType::RSSI_RC_CHANNEL_VALUE :
             receiver_rssi = read_channel_rssi();
             break;
+        case RssiType::RSSI_RECEIVER : {
+            int16_t rssi = hal.rcin->get_rssi();
+            if (rssi != -1) {
+                receiver_rssi = rssi / 255.0;
+            }
```

Comment:
> I think it's important that the user get to define exactly what the RSSI range is for them. This allows us to be flexible and not assume anything about a users use case.

that makes sense if we have any receiver protocols where we can't control the range. This patch only supports SBUS, SUMD and S24, which I think are all 0..255 (although I haven't actually tested SUMD and S24 for RSSI).
Note that the interface from px4io defines it as being 0..255,